examples/PieChartExample.tsx

```diff
@@ -2,46 +2,142 @@ import DonutChart from "../chartsLibrary/SolidDonutChart";
 import SolidPieChart from "../chartsLibrary/SolidPieChart";
 
 const data = [
```

Comment:
Please restore what was in this component. This was not supposed to be edited.

Reply:
I did not edit anything here, dad. This was the same in my VS code.